experiment_pages/experiment/experiment_buttons.py

```diff
@@ -0,0 +1,175 @@
+# pylint: skip-file
```

Comment:
Because of the experiment_menu_ui.py file already importing every other page as their own class (because that is how this entire project is set up), I understand why this file is necessary. However, it causes me pain that is has to exist this way.